### PR TITLE
Fix sidebar hover functionality

### LINKED ISSUE #1: Inklapbaar side menu na inloggen
Als ik ben ingelogd dan wil ik dat het sidemenu automatisch ingeklapt is en dat je alleen de icoontjes ziet. Bij een hover over de icoontjes wil ik dat je de menuitems met tekst ziet. Vervolgens als je weg hovert of weg klikt, dan moeten alleen de icoontjes getoond worden.

### PR DESCRIPTION
**Pull request by Dennis-Rosenbaum**
Fixes #1

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/Prog-Party/dynamic-qr-frontend/pull/2?shareId=b43157c8-f862-4e51-a453-7060edb1f0ec).

### CODE REVIEW

**Comment by Dennis-Rosenbaum** [comment]
Gesloten: 
image

On hover:
image

Wat een ruimte!

@jensdenbraber zou jij deze pull request willen checken/approven

File: public/styles/globals.css [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+}
+
+.sidebarNav .MuiListItemButton-root .MuiListItemText-root {
+  display: none;
+  margin-left: 10px;
+}
+
+.sidebarNav .MuiListItemButton-root:hover .MuiListItemText-root {
+  display: block;
+} */
```
Comment (jensdenbraber) [inline public/styles/globals.css]: commented code: please remove

File: src/components/layout/sidebar/NavGroup/NavGroup.tsx [hunk 2]
```diff
@@ -24,7 +25,12 @@ const NavGroup = ({ item }: ItemType) => {
     }),
   )
   return (
-    <ListSubheaderStyle>{item.subheader}</ListSubheaderStyle>
+    <ListSubheaderStyle>
+      {props.isCollapsed ? <>
+        {props.item.subheader?.substring(0, 3)}...
+      </> : props.item.subheader}
+      {/* {item.subheader} */}
```
Comment (jensdenbraber) [inline src/components/layout/sidebar/NavGroup/NavGroup.tsx]: commented code: please remove

**Review by jensdenbraber — COMMENTED** [review]
small comments

**Pull request merged**